### ISSUE TITLE
GLK Peer Review QA - Issue 3

### ISSUE THREAD

**Issue by gkerber** [Helen1479/FGHJ4567.github.io#6]
3.
Page Title: Greatest Placement
Filename/URL:   https://helen1479.github.io/FGHJ4567.github.io/index.html
Type of Issue: Validity – Accessibility
Description of Issue:  Page failed accessibility testing:
**Accessibility Insights for Web** 
15 failures were detected:
color-contrast: Elements must have sufficient color contrast (13) WCAG 1.4.3
html-has-lang: <html> element must have a lang attribute (1) WCAG 3.1.1
list: <ul> and <ol> must only directly contain <li>, <script> or <template> elements (1) WCAG 1.3.1
**Wave Web Accessibility Evaluation Tool**
14 contrast errors
